Makefile

```diff
@@ -33,6 +33,8 @@ prepdocs:
 		--env EVALUATION_SYSTEM_CONFIG_FILE $(EVALUATION_SYSTEM_CONFIG_FILE) \
 		--env EVALUATION_SYSTEM_PLUGINS $(EVALUATION_SYSTEM_PLUGINS)
 	python3 -m bash_kernel.install
+	rm -rf docs/source/APIRef.rst
+	curl -Ls -o docs/source/APIRef.rst https://raw.githubusercontent.com/FREVA-CLINT/freva-nextgen/main/docs/source/databrowser/APIRef.rst
```

Comment:
I wanted to ask why we are using the databrowserAPI API reference documents here. IMHO having the databrowserAPI reference, which is included here but is not applicable in Freva, next to the plugin API (which has the `API Reference` header in the Freva documents) might confuse users/admins.  I think after deploying the docs, we have `API Reference` and `Databrowser Rest API` on the header.  just as a minor suggestion.